src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermDiscount = 50;
+  const normalTermDiscount = 20;
+  const rentalCost = 40;
+  const longTerm = 7;
+  const normalTerm = 3;
+  let totalCost = days * rentalCost;
```

Comment:
Consider using `const` instead of `let` for `totalCost` since it's not being reassigned after its initial calculation. This helps to ensure that its value won't be changed unintentionally later in the code.